app/views/app/index.html.haml

```diff
@@ -1,2 +1,4 @@
-= react_component 'ScreensmartApp', {csrfToken: form_authenticity_token},
-                                    class: 'app'
+:javascript
+  window.csrfToken = '#{form_authenticity_token}'
```

Comment:
Tja, uiteindelijk willen we (ik en Arnold) graag van jQuery af, dus het lijkt me zonde om nu werkende code náár jQuery om te schrijven.